summerofcode/2022.md

```diff
@@ -405,6 +405,14 @@ Description: The [in-toto Jenkins plugin](https://github.com/jenkinsci/in-toto-p
 - Mentor(s): Sedef Savas (@sedefsavas) Richard Case (@richcase)
 - Upstream Issue (URL): https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/2178
 
+### Prometheus
+
+#### Add Rule-Length-Encoding as TSDB encoding
```

Comment:
How about this? (And it is Run for R)
```suggestion
#### Add Run-Length-Encoding for samples in TSDB
```

Reply:
lol, that has to be one of my best typos in a long time. :joy: